src/test/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManagerUnitTest.java

```diff
@@ -201,6 +201,20 @@ public void testMappingReadMateRepair() {
         Assert.assertEquals(read2.getMateStart(), 10003);
     }
 
+    @Test
```

Comment:
@jamesemery Have you tested what happens if you add a read with a @s in the name which could recreate the problem?